extensions/Vendor/EXT_animation_map/schema/EXT_animation_map.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "EXT_animation_map extension",
+    "type" : "object",
+    "definitions": {
+        "semanticBinding": {
+            "type": "object",
```

Comment:
I'm not sure how to read these schema definitions so I'll just ask... can a bindings object have `extras`?

Reply:
i believe anything in a gltf schema can have extras right?

Reply:
In that case, `semanticBinding` definition needs to contain
```json
"allOf": [ { "$ref": "glTFProperty.schema.json" } ],
```

Reply:
done, thanks